CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+# Contributing to Celestia
+
+Welcome and Thank You for contributing to building Celestia, the first modular
+blockchain network. 
+
+In this guide you will find information about how Celestia manages the
+`CelestiaOrg` Github as well as expectation around our Engineering Standards.
+
+## Github Management
+
+The `CelestiaOrg` Github uses teams to manage access to the organization and
+its repositories. Members of the Celestia core team and key contributors are
+added to the Github team that aligns with the scope of work they perform. For
+example, those working on the `celestia-node` repository are added to the
+`celestia-node` team. 
+
+### Permissions
+
+By default, all teams have Triage access to all repositories in the
+`CelestiaOrg` Github. This allows for anyone to help manage issues and pull
+request (i.e. adding labels). Teams are given write access to the repositories
+that they are responsible for working on. For example, the `celestia-node` team
+would have write access to the `celestia-node` repository.
+
+### .github
+
+Celestia utilizes the organization's `.github` repository. This repository is
+used to store common organization level content like Github actions, issue
+templates, PR templates, etc.
+
+## Development
+
+### What the Fork
+
+The default development flow is to fork the repository that you are working on
+in order to submit a PR. If you have write access to a repository, because you
+are a member of that sub team, then you can push your development branches
+directly to the repository.
```

Comment:
[proposal] if we encourage pushing development branches directly to the celestiaorg repo, I think we should also encourage repo maintainers to enable this setting

<img width="788" alt="Screen Shot 2022-10-17 at 11 29 05 AM" src="https://user-images.githubusercontent.com/3699047/196219290-ea110063-5c9f-4805-8ab8-5cdd9acb68a5.png">


so that we can avoid instances of https://github.com/celestiaorg/celestia-core/issues/830 . Note that we may still end up with unmerged stale branches on the repo so repo maintainers may need to manually delete stale branches.

Reply:
Yes I plan to have a list of repo setting in here as well and will include that one for sure.